.gitmodules

```diff
@@ -1,3 +0,0 @@
-[submodule "ros_ws/src/external_packages/vesc"]
-	path = ros_ws/src/external_packages/vesc
-	url = https://github.com/mit-racecar/vesc.git
```

Comment:
Can you just change this URL to https://github.uconn.edu/ret15108/vesc instead of copying the code?

Reply:
...Pretty much. I forgot that the new one should be a submodule as well :/